docs/rewards/reward-boosts.mdx

```diff
@@ -35,8 +35,22 @@ This is the percentage of the 3M tokens that correspond to the rewardable hours
 
 `DailyBetaRewards[s] = StationBetaRewards[s]/365`: This is the beta rewards allocated daily to this station.
 
+### Beta Rewards Dilution - A fix
 
+Initially, the reward allocation formula caused dilution as the launch day was delayed, disadvantaging long-term station owners. 
 
+#### Updated Beta Rewards Allocation 
+
+The updated approach involves calculating beta rewards twice: once at the end of February 2024 and again just of the project's beta phase (May 27 2024) . 
```

Comment:
and again just of the project's beta phase (May 27 2024) . ----> and again just at the end of the project's beta phase (May 27 2024).